packages/components/atoms/f-image-tile/stories/TestContainer.vue

```diff
@@ -16,7 +16,7 @@
 import ImageTile from '../src/components/ImageTile.vue';
 
 export default {
-    name: 'Container',
+    name: 'TestContainer',
```

Comment:
The `name` property is already generated automatically, do we need a rule to enforce adding the property manually?

Reply:
Same as above – the linting started picking up on all the one-word components (like this one). This was the safest way I could think of to fix them until people go in and actually update their components to be two word descriptors as is recommended in the vue docs